src/network/network.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <memory>
+#include "common/assert.h"
+#include "common/logging/log.h"
+#include "enet/enet.h"
+#include "network/network.h"
+
+namespace Network {
+
+std::unique_ptr<RoomMember> g_room_member; ///< RoomMember (Client) for network games
+std::unique_ptr<Room> g_room;              ///< Room (Server) for network games
+
+bool Init() {
+    ASSERT_MSG(enet_initialize() == 0, "Error initalizing network");
+    g_room = std::make_unique<Room>();
+    g_room_member = std::make_unique<RoomMember>();
+    LOG_DEBUG(Network, "initialized OK");
+    return true;
+}
+
+Room* getRoom() {
```

Comment:
I'm not sure I like these functions, after all. They kind of defeat the purpose of using unique_ptr, maybe just return a reference to the unique_ptr